Core/DISET/private/InnerRPCClient.py

```diff
@@ -58,7 +58,10 @@ def executeRPC( self, functionName, args ):
             return retVal
 
       # Send the arguments to the function
-      retVal = transport.sendData( S_OK( args ) )
+      # Note: we need to convert the arguments to list
+      # We do not need to deseralize it because variadic functions
```

Comment:
what's `variadic`?

Reply:
`*args, **kwargs` 
but do not bother to review yet, it's far from complete and will be reshuffled